app/controllers/project_controller.rb

```diff
@@ -0,0 +1,23 @@
+class ProjectController < ApplicationController
+  default_form_builder GOVUKDesignSystemFormBuilder::FormBuilder
```

Comment:
It might be worth popping this into `ApplicationController` so it is the default everywhere.